From 34346a8ac129047c8ba345562a78c74a9bae8096 Mon Sep 17 00:00:00 2001
From: Julien Schueller <schueller@phimeca.com>
Date: Tue, 23 May 2017 09:33:45 +0200
Subject: [PATCH] Do not require typing module for python>=3.5

pkg_resources.DistributionNotFound: The 'typing' distribution was not found and is required by Sphinx
---
 setup.py | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 95bb3778f..4544ca3b0 100644
--- a/setup.py
+++ b/setup.py
@@ -51,10 +51,14 @@
     'alabaster>=0.7,<0.8',
     'imagesize',
     'requests>=2.0.0',
-    'typing',
     'setuptools',
     'sphinxcontrib-websupport',
 ]
+
+# this is a backport to versions older than 3.5
+if sys.version_info < (3, 5):
+    requires.append('typing')
+
 extras_require = {
     # Environment Marker works for wheel 0.24 or later
     ':sys_platform=="win32"': [
